vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -110,3 +111,13 @@ def callback_update_theme(on: bool):
     @staticmethod
     def _create_error_page_404():
         return dash.register_page(module=MODULE_PAGE_404, layout=create_layout_page_404())
+
+    @staticmethod
+    def _build_actions():
+        from vizro.models import Action
+
+        return [
+            action_component
+            for _, action in model_manager._items_with_type(Action)
+            for action_component in action.build()
+        ]
```

Comment:
This would optimally live in the ActionManager for me. And the Dashboard would then just call `ActionManager._build_actions()` instead of this being a `staticmethod` of the Dashboard. As this is not really the responsibility of the Dashboard.